Makefile

```diff
@@ -26,6 +26,23 @@ securedrop-workstation-grsec: ## Builds Debian metapackage for Qubes Workstation
 install-deps: ## Install initial Debian packaging dependencies
 	./scripts/install-deps
 
+.PHONY: requirements
+requirements: ## Creates requirements files for the Python projects
+	./scripts/create-requirements
+	./scripts/update-requirements
+
+.PHONY: build-wheels
+build-wheels: syncwheels ## Builds the wheels and syncs to the localwheels directory
+	./scripts/build-sync-wheels -r ${PKG_DIR}/requirements.txt
```

Comment:
so we don't need the requirements*.txt files in any of the proxy or client repos if this is merged, correct?

Reply:
or do we use this script to generate those files and then commit them in the upstream repository? we should document this in the README if so

Reply:
requirements*.txt files will be stored in the git and also be part of the source release. The pip will use these files to do the final install for the debian package. I will update the guide and readme for this.